### ISSUE TITLE
Arrows are disabling, but it is not yet the last slide

### ISSUE THREAD

**Issue by jonasmarco** [nolimits4web/swiper#4583]
- Swiper Version: ^6.4.7
- Platform/Target and Browser Versions: Windows, any browser

## What You Did

I am trying to implement a series of slides. However, I am facing a bug that I had not had until today. Swiper disables the forward arrow in the middle of the slides, before reaching the last slide item of the carousel.

**The code is basically this**
https://github.com/jonasmarco/swiper-issue

**Here you can see an example of the bug**
http://sites.clickweb.com.br/paranatex.com.br/produto/3/shibori
http://sites.clickweb.com.br/paranatex.com.br/produto/1/mata-atlantica

**Here you can see an WORKING example**
http://sites.clickweb.com.br/paranatex.com.br/produto/6/shibori-

## Expected Behavior

Basically, the Swiper does not deactivate the arrows before reaching the last item or vice versa, when returning from the last to the first. If you need more details I can provide it.
